frontend/src/Components/Organisms/TagsTable/TagsTable.tsx

```diff
@@ -0,0 +1,61 @@
+import { useEffect, useState } from "react";
+import { Post } from "_API/Types/Post";
+import { Tag } from "_API/Types/Tag";
+import Table from "_Components/Molecules/Table/Table";
+import { getTagData } from "_Services/SyncService";
+
+const TagsTable = () => {
+  const [tags, setTags] = useState<Tag[]>([]);
+  const [loaded, setLoaded] = useState(false);
+  
+  useEffect(() => {
+    const loadTagData = async () => {
+      let tagViewModels = await getTagData();
+      setTags(tagViewModels);
+    };
+    loadTagData();
+  }, []);
+
+  useEffect(() => {
+    if (tags) setLoaded(true);
+  }, [tags]);
+
+  const RenderTagRow = (tag: Tag) => (
```

Comment:
I don't see any need for this to be a separate function, can we include the JSX that this function returns in the one place where it's called